build-output-api/on-demand-isr/.vercel/output/functions/index.func/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (req, res) => {
+  res.setHeader('Content-Type', 'text/html; charset=utf-8')
+
+  res.end(`
+<h1>On-demand Incremental Static Regeneration (ISR) Example</h1>
+<p>The "server time" below will only get updated after you click the Revalidate link.</p>
+<p>Server time: ${new Date().toISOString()}</p>
+<p><a href="/revalidate">Revalidate</a></p>
```

Comment:
I don't think we should demonstrate this via an open-link, we need to stress/demonstrate that this needs to be behind some kind of authentication